ansible/dynhosts

```diff
@@ -188,6 +188,15 @@ def main():
         elif args.host:
             output = to_json(inventory.get_host(args.host))
         print(output)
+
+        # Write inventory file in /etc/ansible/hosts
+        with open("/etc/ansible/hosts", "w+") as hostfile:
+            hostfile.write('#!/usr/bin/env python\n\n')
```

Comment:
cant you write direct json / yaml file?
https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html#hosts-and-groups

Reply:
Just checked out, we actually can specify a json file as extra vars. doing..

Reply:
done